### PR TITLE
OPSEXP-1862 Add alfresco repository chart

### PR DESCRIPTION
**Pull request by gionn**
Ref: OPSEXP-1862

- [ ] https://github.com/Alfresco/acs-deployment/issues/930

### CODE REVIEW

File: charts/alfresco-repository/templates/_helpers-name.tpl
Comment (gionn) [inline charts/alfresco-repository/templates/_helpers-name.tpl:1]: please review first this

File: charts/alfresco-repository/templates/_helpers-label.tpl
Comment (gionn) [inline charts/alfresco-repository/templates/_helpers-label.tpl:1]: please review first this approach

**Review by alxgomz — COMMENTED** [review]
The naming system will indeed trigger a lot of changes in the other charts to be aligned.
However I think by being conservative we would be sticking too much to what we have today and we no is not optimal..
So that's ok for me to proceed with this new approach.
We'll probably need a dedicated Eepic for aligning charts...

**Comment by gionn** [comment]
I will wait for https://github.com/Alfresco/acs-deployment/pull/902 and integrating the changes before continuing but worth another round of review

**Review by alxgomz — COMMENTED** [review]
Mostly commented on the naming logic

File: charts/alfresco-repository/values.yaml [hunk 1]
```diff
@@ -0,0 +1,397 @@
+repository:
```
Comment (alxgomz) [inline charts/alfresco-repository/values.yaml:1]: That level of values should be removed and its content moved to the root context.

**Review by alxgomz — COMMENTED** [review]
Beyond the comment, I am concerned the chart still contains a lot of logic that's inhereted from `alfresco-content-services` chart and shoudl only be present in the parent chart.
For example, all the logic related to configuring a service based on wether it's in-cluster or not (`activemq` vs `messageBroker`, `database` vs `postgresql`)

**Comment by gionn** [comment]
Deprecated by https://github.com/Alfresco/alfresco-helm-charts/pull/84

**Pull request closed without merging**